src/OXRS_LCD.h

```diff
@@ -8,6 +8,7 @@
 #include "roboto_fonts.h"
 #include <Ethernet.h>
 #include <WiFi.h>
+#include <SPIFFS.h>
```

Comment:
I am never sure if it is better to have all `#includes` in the header, or only have the ones needed by the header and move the others into the program file. What do you think?

For the other libraries I have been doing the latter - so if there is no reference to a `SPIFFS` class or constant, I move it to the program file.

Reply:
Change this include to;

```
#if defined(ARDUINO_ARCH_ESP32)
#include <SPIFFS.h>
#elif defined(ARDUINO_ARCH_ESP8266)
#include <FS.h>
#endif
```

So it will run on an ESP8266 if required.